src/components/StatsLeaderboard.vue

```diff
@@ -13,14 +13,46 @@
         v-model="selectedWeeks"
         :items="weeks"
         label="Select Weeks"
+        class="week-select"
         data-cy="week-select"
-        class="flex-grow-1"
-        menu-props="data-week-select-menu"
         multiple
       />
     </div>
+    <v-table>
+      <!-- Headers -->
+      <tr>
+        <th v-for="header in tableColumns" :key="header.value">
+          {{ header.text }}
+        </th>
+      </tr>
+      <!-- Body -->
+      <tbody>
+        <tr v-for="row in tableRows" :key="row.username" :class="tableRowClass(row)">
+          <!-- Username -->
+          <td :data-username="row.username">
+            {{ row.username }}
+          </td>
+          <!-- Rank -->
+          <td :data-rank="row.username">
+            {{ row.rank }}
+          </td>
+          <td v-for="(week) in ['total', ...selectedWeeks]" :key="`${row.username}-${week}`">
+            <!-- {{ row[column.value] ?? '' }} -->
+            <stats-leaderboard-cell
+              :player-row="row"
+              :week="week"
+              :selected-metric="selectedMetric"
+              :players-beaten="playersBeaten(row.username, week)"
+              :players-lost-to="playersLostTo(row.username, week)"
+              :top-total-scores="topTotalScores"
+              :season-name="seasonName"
+            />
+          </td>
+        </tr>
+      </tbody>
+    </v-table>
     <!-- https://vuetifyjs.com/en/api/v-data-table/ -->
-    <v-table
+    <!-- <v-data-table
```

Comment:
Remove this whole block, we have the git history and don't need it in the template.

Reply:
Oops 😅